Makefile

```diff
@@ -13,11 +13,9 @@ ghdist:
 	cd dist && git checkout gh-pages
 
 publish:
-	./node_modules/gulp/bin/gulp.js clean
-	cd dist && git pull origin gh-pages
-	./node_modules/gulp/bin/gulp.js publish
+	./node_modules/gulp/bin/gulp.js build
 	echo -n "Please open http://localhost:8000/dist/index.html and check if everything works fine." && read -e
-	cd dist && git add --all . && git commit -am "merged with master" && git push origin gh-pages
+	./node_modules/gulp/bin/gulp.js deploy
```

Comment:
In theory `ghdist` and `publish` could be removed and you could simply use `gulp deploy` to deploy to gh-pages.